app/src/main/java/com/asfoundation/wallet/ui/gamification/MyLevelFragment.kt

```diff
@@ -91,16 +110,33 @@ class MyLevelFragment : DaggerFragment(), MyLevelView {
       } else {
         R.string.gamification_how_table_b2
       }
-      gamification_progress_bar.setLevelBonus(levelIndex,
-          getString(bonusLabel, gamification_progress_bar.formatLevelInfo(value)))
+
+      if (pioneer) {
+        gamification_progress_bar_pioneer.setLevelBonus(levelIndex,
+            getString(bonusLabel, gamification_progress_bar_pioneer.formatLevelInfo(value)))
+      } else {
+        gamification_progress_bar_normal.setLevelBonus(levelIndex,
+            getString(bonusLabel, gamification_progress_bar_normal.formatLevelInfo(value)))
```

Comment:
Why do we need 2 separate views? isn't the setPioneer method enough?

Reply:
The development started that way. But at some point in time the code was too confusing. BEcause of that I've decided to split in 2 layouts